ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -728,16 +728,40 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent keyEvent) {
             // Any 'Enter' action will do
             if ((actionId & EditorInfo.IME_MASK_ACTION) > 0 ||
```

Comment:
Seems that we also have to exclude `IME_ACTION_NEXT` and `IME_ACTION_PREVIOUS` here.